src/main/java/hudson/plugins/analysis/core/AbstractResultAction.java

```diff
@@ -51,7 +53,7 @@
      * @param result
      *            the result of the action
      */
-    public AbstractResultAction(final AbstractBuild<?, ?> owner, final AbstractHealthDescriptor healthDescriptor, final T result) {
+    public AbstractResultAction(final Run<?, ?> owner, final AbstractHealthDescriptor healthDescriptor, final T result) {
```

Comment:
Oh never mind, you did—just in a different place in the source file than I expected it.